docs/dsp-tools-usage.md

```diff
@@ -254,61 +254,45 @@ In order to upload data incrementally the procedure described [here](dsp-tools-x
 
 
 
-## Start a DSP-stack on your local machine (for DaSCH-internal use only)
+## Start a DSP stack on your local machine 
+
+DSP-API is the heart of the DaSCH software - it is a server application for storing data from the Humanities.
+DSP-APP is our generic user interface, i.e. the viewer to look at the data. It's a server application, too.
 
 For testing purposes, it is sometimes necessary to run DSP-API and DSP-APP on a local machine. But the startup 
 and shutdown of API and APP can be complicated: Both repos need to be cloned locally, a `git pull` has to be executed 
-from time to time to stay up to date, and then there are several commands for each repository to remember. 
-
-Another challenge is the software that DSP depends upon: JDK, node, npm, Angular, etc. should be kept up to date. And
-it might happen that a dependency is replaced, e.g. JDK 11 Zulu by JDK 17 Temurin. An non-developer can quickly get lost
-in this jungle. 
-
-That's why dsp-tools offers some commands to facilitate the handling of API and APP. These commands 
-
- - clone the repos to `~/.dsp-tools`, and keep them up to date.
- - check every time if the dependencies are up to date, and give you advice how to update them, if necessary.
- - pass on the right commands to APP and API, even if the correct usage of these commands changes over time.
- - make sure that the repos don't get cluttered with old files over time.
- - log their activity in `~/.dsp-tools`, so you can check the logs for troubleshooting, if necessary.
-
-The only requirements for these commands are:
-
- - the initial installation of all software that you accomplished when you started working at DaSCH
- - Docker must be running (for DSP-API only)
-
-Please note that these commands were developed for DaSCH-internal use only. They only work on Macs that have the 
-required software installed that makes it possible to run the API and APP. We don't offer support or troubleshooting 
-for these commands.
-
+from time to time to stay up to date, and then there are several commands for each repository to remember. And your 
+local clone of the API might get cluttered with old files over time.
 
-### Start DSP-API
+Another challenge is the software that DSP depends upon, and that should be kept up to date: JDK, node, npm, Angular, 
+etc. And it might happen that a dependency is replaced, e.g. JDK 11 Zulu by JDK 17 Temurin. A non-developer can quickly 
+get lost in this jungle. 
 
+That's why dsp-tools offers a command to facilitate the handling of API and APP:
 ```
-dsp-tools start-api
+dsp-tools start-stack
```

Comment:
maybe I'm professionally deformed by our make targets... but to me `dsp-tools stack-up` (or even shorter `dsp-tools stack`) and `stack-down` would sound more familiar/intuitive.

Reply:
I wanted to find a terminology that prevents confusion with the make-targets, otherwise I confuse them. In the initial phase, when the two were similar, I accidentally typed things like `dsp-tools stack-down-delete-volumes`.